sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -122,8 +128,8 @@
     },
     "/configurations": {
       "get": {
-        "summary": "Get multiple configurations for devices or modules of an IoT Hub. Returns the specified number of configurations for Iot Hub. Pagination is not supported.",
-        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "summary": "Get multiple configurations for Iot Hub devices or modules.",
+        "description": "Get multiple configurations for Iot Hub devices or modules.",
```

Comment:
Does ADM work on edge? or it specific only to IotHub?

Reply:
We can have ADM for both devices and modules